front-end/src/components/FileStorage.js

```diff
@@ -66,9 +61,12 @@ const CustomFileRenderer = (props) => {
 };
 
 const CustomFolderRenderer = (props) => {
-    const { isOpen, browserProps } = props;
+    const { files, isOpen, browserProps } = props;
```

Comment:
There is a better way than passing in all files. The package used already passes down all the information you need in props, try printing that to console and check it out. No need to pass additional stuff here on top of isOpen, browserProps